server/integration/infinispan/src/main/java/org/jboss/as/clustering/infinispan/task/DeployedTaskEngine.java

```diff
@@ -0,0 +1,43 @@
+package org.jboss.as.clustering.infinispan.task;
```

Comment:
I'm not sure if this is the best place for Task Engine. Maybe @tristantarrant has some thoughts...